virtualfish/__main__.py

```diff
@@ -5,7 +5,11 @@
 
 
 if __name__ == "__main__":
-    version = pkg_resources.get_distribution('virtualfish').version
+    try:
+        version = pkg_resources.get_distribution('virtualfish').version
+    except:
```

Comment:
I'd prefer it if you caught a specific exception here instead of using a bare `except` block.